berkeley-mobile/StudyPact/StudyPact.swift

```diff
@@ -115,10 +115,58 @@ extension StudyPact {
             }
         }
     }
-
+    
     // MARK: GetUser
-
+    
+    public func getUser(completion: @escaping ([String : Any]?) -> Void,
+                        errorCompletion: ((Error) -> Void)? = nil) {
+        guard let cryptohash = self.cryptoHash,
+              let email = self.email,
+              let url = EndpointKey.getUser.url else {
+            completion(nil)
+            return
+        }
+        let params = ["user_email": email, "secret_token": cryptohash]
+        NetworkManager.shared.get(url: url, params: params) { response in
+            switch response {
+            case .success(let data):
+                completion(data)
+            case .failure(let error):
+                errorCompletion?(error)
+            }
+        }
+    }
+    
     // MARK: AddUser
+    
+    public func addUser(name: String, email: String, phone: String?, profile: String?, facebook: String?, completion: @escaping (Bool) -> Void) {
```

Comment:
<img width="630" alt="Screen Shot 2021-02-11 at 4 04 52 PM" src="https://user-images.githubusercontent.com/41145903/107714912-f82a4480-6c82-11eb-9115-aedbe015c843.png">
Getting this when registering under a new email, unsure if it's just on my end / backend / whatever.

Reply:
Most likely a problem with backend, seems like the demo email is also running into this issue.